### PR TITLE
Adds a big fat mouse.

### PR DESCRIPTION
**Pull request by Earthcrusher**
I saw [this](https://github.com/VOREStation/VOREStation/issues/2022) and pretty much had to do this immediately. It's like I have no impulse control whatsoever.

The boy:
image

### CODE REVIEW

**Comment by Zuhayr** [comment]
But why

**Comment by Earthcrusher** [comment]
I mean. Did you not see the image?

**Comment by Zuhayr** [comment]
I saw it but I am not clear on why we need a giant vore mouse. I guess it's only in the cide, not spawned anywhere.

**Comment by Earthcrusher** [comment]
omg leave him alone, he's just a chubber, he doesn't go in for that fetish shit :(

**Comment by Zuhayr** [comment]
Uhh.
image

**Comment by SpadesNeil** [comment]
http://www.fatmouse.org/ this is why just roll with it

**Comment by SpadesNeil** [comment]
image

See? Not a vore thing, just a thing we took because it was hilarious. This is canon to the fat mouse.

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -36,6 +36,15 @@
 	can_pull_size = TINY_ITEM
 	can_pull_mobs = MOB_PULL_NONE
 
+/mob/living/simple_animal/mouse/New()
+	if(prob(2))
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: You need to call `..()` when overriding `New()`. `..()` is the DM equivalent of [super](https://stackoverflow.com/questions/5141344/meaning-of-super-keyword).

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -134,4 +143,12 @@
 	..()
 	// Change my name back, don't want to be named Tom (666)
 	name = initial(name)
-	real_name = name
\ No newline at end of file
+	real_name = name
+
+/mob/living/simple_animal/mouse/fat
```
Comment (Zuhayr) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: Unless I'm misreading, fatmouse has a 2% chance to be remade in the image of fatmouse, despite already being a fat mouse.

**Comment by Screemonster** [comment]
**FATMOUSE**, as you might tell from the style of the webpage, is an old web 1.0 meme. It's probably older than some players.

**Pull request closed without merging**